internal/distro/distro_test.go

```diff
@@ -20,8 +21,11 @@ func TestDistro_Pipeline(t *testing.T) {
 		t.Errorf("Could not read pipelines directory '%s': %v", pipelinePath, err)
 	}
 	for _, fileInfo := range fileInfos {
+		if !strings.HasSuffix(fileInfo.Name(), "empty_blueprint.json") {
```

Comment:
Hm, this will break as soon as we introduce other blueprint tests. If the given blueprint doesn't currently produce the expected pipeline, then I'd just remove it (maybe put it in the commit message for documentation purposes?). The blueprint tests should ignore any files with missing composes.

Reply:
fixed by renaming the json object containing metadata about the test